xero/auth.py

```diff
@@ -326,26 +324,20 @@ class PartnerCredentials(PublicCredentials):
 
         >>> rsa_key = "-----BEGIN RSA PRIVATE KEY----- ..."
 
-     2) client_cert is no longer used regarding new Xero security policy, 
-        but stays for now for backward-compatibility.
-        
-        >>> client_cert = ('/path/to/entrust-cert.pem',
-                           '/path/to/entrust-private-nopass.pem')
-
-     3) Once a token has expired, you can refresh it to get another 30 mins
+     2) Once a token has expired, you can refresh it to get another 30 mins
 
         >>> credentials = PartnerCredentials(**state)
         >>> if credentials.expired():
                 credentials.refresh()
 
-     4) Authorization expiry and token expiry become different things.
+     3) Authorization expiry and token expiry become different things.
 
         oauth_expires_at tells when the current token expires (~30 min window)
 
         oauth_authorization_expires_at tells when the overall access
         permissions expire (~10 year window)
     """
-    def __init__(self, consumer_key, consumer_secret, rsa_key, client_cert=None,
+    def __init__(self, consumer_key, consumer_secret, rsa_key,
```

Comment:
I think you need to add **kwargs here so we don't break backwards compatability?